week2/basicsorting.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <script>
+    let arraytosort = [];
+    for(let i = 0; i < arraytosort.length; i++){
```

Comment:
Sorting will require two loops to completely do it. If you only use one loop it will only sort the numbers once.